src/components/fileUpload.js

```diff
@@ -285,6 +457,7 @@
       span: {
         flex: 1,
         textAlign: 'start',
+        marginBottom: '0.1875rem!important',
```

Comment:
On the next line the same kind of styling is applied, but it is written in an array. Can you choose either one an make them consistent? :)

Reply:
I'll change them to string interpolation, as the array notation wasn't working as done on the next line.